web_environment_ribbon/static/src/css/ribbon.css

```diff
@@ -14,7 +14,7 @@
     -moz-transform: rotate(-45deg);
     -o-transform: rotate(-45deg);
     transform: rotate(-45deg);
-    z-index: 1000;
+    z-index: 9999;
```

Comment:
If your concern is about clicking on the menu through the ribbon, I just added the "pointer-events: none" CSS property, to allow that.